include/maths/time_series/CTimeSeriesModel.h

```diff
@@ -678,6 +681,9 @@ class MATHS_TIME_SERIES_EXPORT CMultivariateTimeSeriesModel : public common::CMo
     //! Unpack the weights in \p weights.
     static TDouble10VecWeightsAry unpack(const TDouble2VecWeightsAry& weights);
 
+    //! Apply time shift at the time port \p time by \p shift amount of seconds.
```

Comment:
There are still a few of these typos (if you search for port \p)
```suggestion
    //! Apply time shift at the time \p time by \p shift amount of seconds.
```